### PR TITLE
enable metadata lock by default

### PR DESCRIPTION
**Pull request by wjhuang2016**
Signed-off-by: wjhuang2016 <huangwenjun1997@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: metadata-lock.md [hunk 1]
```diff
@@ -37,7 +33,7 @@ summary: 介绍 TiDB 中元数据锁的概念、原理、实现和影响。
 
 ## 使用元数据锁
 
-使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 启用或者关闭元数据锁特性。
+在 v6.5.0 及之后的版本中，TiDB 默认开启元数据锁特性。当集群从旧版本升级到 v6.5.0 及之后的版本后，会自动打开元数据锁功能。如果需要关闭元数据锁，可以使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 关闭元数据锁特性。
```
Comment (Oreoxmt) [inline metadata-lock.md:36]:
```suggestion
在 v6.5.0 及之后的版本中，TiDB 默认开启元数据锁特性。当集群从 v6.5.0 之前的版本升级到 v6.5.0 及之后的版本时，TiDB 会自动开启元数据锁功能。如果需要关闭元数据锁，你可以使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 关闭元数据锁。
```

Comment (Oreoxmt) [reply metadata-lock.md:36]: Please remove [the experimental feature in line 38](https://github.com/wjhuang2016/docs-cn/blob/mdl_enable_default/experimental-features.md?plain=1#L38) and modify the default value in [line 68](https://github.com/wjhuang2016/docs-cn/blob/mdl_enable_default/sql-statements/sql-statement-commit.md?plain=1#L68).

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
@wjhuang2016 Could you please invite a tech review? Thanks

File: experimental-features.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
 + [表级锁 (Table Lock)](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入)（v4.0.0 实验特性）
-+ [元数据锁](/metadata-lock.md)（v6.3.0 实验特性）
++ [元数据锁](/metadata-lock.md)
```
Comment (Oreoxmt) [inline experimental-features.md:38]:
```suggestion
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-commit.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ Query OK, 0 rows affected (0.01 sec)
 
 ## MySQL 兼容性
 
-* 目前，默认情况下 TiDB 不使用元数据锁 (MDL) 来防止 DDL 语句修改事务使用的表。如果表定义被更改，提交事务将导致报错 `Information schema is changed`。此时，事务会自动回滚。
+* 目前，默认情况下 TiDB 使用元数据锁 (MDL) 来防止 DDL 语句修改事务使用的表，这与 MySQL 中的元数据在行为上有一些差别，具体可以参考[元数据锁](/metadata-lock.md)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-commit.md:68]:
```suggestion
* 目前，默认情况下 TiDB 使用元数据锁 (MDL) 来防止 DDL 语句修改事务使用的表，这与 MySQL 中的元数据锁在行为上有一些差别，具体可以参考[元数据锁](/metadata-lock.md)。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: metadata-lock.md [hunk 4]
```diff
@@ -37,7 +33,7 @@ summary: 介绍 TiDB 中元数据锁的概念、原理、实现和影响。
 
 ## 使用元数据锁
 
-使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 启用或者关闭元数据锁特性。
+在 v6.5.0 及之后的版本中，TiDB 默认开启元数据锁特性。当集群从 v6.5.0 之前的版本升级到 v6.5.0 及之后的版本时，TiDB 会自动开启元数据锁功能。如果需要关闭元数据锁，你可以使用系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 关闭元数据锁。
```
Comment (Oreoxmt) [inline metadata-lock.md:36]:
```suggestion
在 v6.5.0 及之后的版本中，TiDB 默认开启元数据锁特性。当集群从 v6.5.0 之前的版本升级到 v6.5.0 及之后的版本时，TiDB 会自动开启元数据锁功能。如果需要关闭元数据锁，你可以将系统变量 [`tidb_enable_metadata_lock`](/system-variables.md#tidb_enable_metadata_lock-从-v630-版本开始引入) 设置为 `OFF`。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**